drift/lib/src/runtime/query_builder/migration.dart

```diff
@@ -288,6 +288,21 @@ class Migrator {
       context.buffer.write(')');
     }
 
+    if (table.uniqueKeys.isNotEmpty) {
```

Comment:
Can you also add a test to `drift` by adding a unique key to some table in `test/data/tables/todos.dart` and make sure that the unique constraint is generated in the migrator?

Reply:
Sure!